service/src/main/java/uk/gov/hmcts/reform/fpl/controllers/ManageOrderPostSubmitController.java

```diff
@@ -0,0 +1,41 @@
+package uk.gov.hmcts.reform.fpl.controllers;
+
+import io.swagger.annotations.Api;
+import lombok.RequiredArgsConstructor;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+import uk.gov.hmcts.reform.ccd.client.model.AboutToStartOrSubmitCallbackResponse;
+import uk.gov.hmcts.reform.ccd.client.model.CallbackRequest;
+import uk.gov.hmcts.reform.ccd.client.model.CaseDetails;
+import uk.gov.hmcts.reform.fpl.service.orders.ManageOrderDocumentScopedFieldsCalculator;
+import uk.gov.hmcts.reform.fpl.service.orders.ManageOrdersEventBuilder;
+import uk.gov.hmcts.reform.fpl.service.orders.OrderProcessingService;
+import uk.gov.hmcts.reform.fpl.service.orders.prepopulator.modifier.ManageOrdersCaseDataFixer;
+
+import java.util.Map;
+
```

Comment:
Why do we need a class for this? Should we maybe include in the controller like we do for the gatekeeping order stuff.

Reply:
We don't need a new class for this. I moved the endpoint to ManageOrdersController just like the gatekeeping order.